src/main/java/org/elasticsearch/indices/memory/IndexingMemoryController.java

```diff
@@ -113,13 +157,14 @@ public IndexingMemoryController(Settings settings, ThreadPool threadPool, Indice
             translogBuffer = ByteSizeValue.parseBytesSizeValue(translogBufferSetting, null);
```

Comment:
Could you replace null above with `TRANSLOG_BUFFER_SIZE_SETTING`?  (This is a separate issue, but I never backported to 1.7.x...)